src/Authentication/Services/SystemUserService.cs

```diff
@@ -104,10 +106,12 @@ public async Task<List<SystemUser>> GetListOfSystemUsersForParty(int partyId)
         /// Set the Delete flag on the identified SystemUser
         /// </summary>
         /// <returns>Boolean True if row affected</returns>
-        public async Task<bool> SetDeleteFlagOnSystemUser(Guid systemUserId)
+        public async Task<bool> SetDeleteFlagOnSystemUser(string partyId, Guid systemUserId, CancellationToken cancellationToken = default)
         {
-            await _repository.SetDeleteSystemUserById(systemUserId);            
-
+            await _repository.SetDeleteSystemUserById(systemUserId);
+            SystemUser systemUser = await _repository.GetSystemUserById(systemUserId);
+            List<Right> rights = await systemRegisterService.GetRightsForRegisteredSystem(systemUser.SystemId, cancellationToken);
```

Comment:
Som vi snakket om på Grooming i dag, så blir dette egentlig feil. Men fordi AM ikke har laget API for å spørre etter hva som er delegert til SystemBruker ennå (mars?) så er det greit for nå. Men vi må notere at dette skal endres senere!